rusk-wallet/src/bin/interactive.rs

```diff
@@ -208,15 +210,26 @@ enum AddrOp {
 
 #[derive(PartialEq, Eq, Hash, Clone, Debug)]
 enum CommandMenuItem {
+    // phoenix
+    StakeInfo,
```

Comment:
Isn't `StakeInfo` agnostic to the transaction model used, since it doesn't create a transaction but only queries the state? If yes, we shouldn't add it under the phoenix comment but somewhere below

Reply:
@moCello Moonlight staking doesn't seem to work for me. I'm getting the "Expected Bls Public Keyition..." message.

Phoenix staking with this PR seems to have a range issue on how much you can provide as value. 

StakeInfo should be agnostic though.